### PR TITLE
Update some grammars to a commit where the license file is included

### PR DESCRIPTION
**Pull request by blinxen**
This is the second part of https://github.com/helix-editor/helix/pull/8691.
This PR should include all remaining grammars, that don't have license files. Except `tree-sitter-eex` because upstream seems to be dead.

I am unsure if the changes to the `scm` files are 100% OK. I'd appreciate it if someone could double check them.

### CODE REVIEW

**Comment by blinxen** [comment]
Thanks for the review! I applied your suggested changes.

File: runtime/queries/pod/highlights.scm [hunk 1]
```diff
@@ -1,61 +1,72 @@
...
-  (content) @markup.heading.4)
-(head_paragraph
-  (head_directive) @directive
-  (#eq? @directive "=head5")
-  (content) @markup.heading.5)
-(head_paragraph
-  (head_directive) @directive
-  (#eq? @directive "=head6")
-  (content) @markup.heading.6)
-
```
[the remark this answers is not in the document]
Comment (the-mikedavis) [reply runtime/queries/pod/highlights.scm:33]: This is not yet resolved: the pattern on L7 should match `=head1`, `=head2`, etc. and use the `markup.heading.1`, `markup.heading.2`, etc. captures

File: runtime/queries/pod/highlights.scm [hunk 2]
```diff
@@ -1,61 +1,72 @@
...
+
+(command_paragraph
+  (command) @keyword
+  (#match? @keyword "^=head")
+  (content) @markup.title)
+
+(command_paragraph
+  (command) @keyword
+  (#match? @keyword "^=over")
+  (content) @number)
```
Comment (the-mikedavis) [inline runtime/queries/pod/highlights.scm:15]:
```suggestion
  (content) @constant.numeric)
```

File: runtime/queries/pod/highlights.scm [hunk 3]
```diff
@@ -1,61 +1,72 @@
...
+(command_paragraph
+  (command) @keyword
+  (#not-match? @keyword "^=(head|over|item|encoding)")
+  (content) @string)
 
 (verbatim_paragraph (content) @markup.raw)
 
-(interior_sequence) @tag
+(interior_sequence
+  (sequence_letter) @character
```
Comment (the-mikedavis) [inline runtime/queries/pod/highlights.scm:35]:
```suggestion
  (sequence_letter) @constant.character
```

There are a few instances of this that can be changed in this file

File: runtime/queries/pod/highlights.scm [hunk 4]
```diff
@@ -1,61 +1,72 @@
...
+  (#eq? @character "X")
+  (content) @markup.reference)
+
 (interior_sequence
-  (sequence_letter) @letter
-  (#eq? @letter "L")
-  (content) @markup.link.url)
+  (sequence_letter) @character
+  (#eq? @character "E")
+  (content) @string.escape)
```
Comment (the-mikedavis) [inline runtime/queries/pod/highlights.scm:72]:
```suggestion
  (content) @string.special.escape)
```

File: runtime/queries/pod/highlights.scm [hunk 5]
```diff
@@ -1,61 +1,72 @@
...
+  (#eq? @character "I")
+  (content) @markup.emphasis)
+
 (interior_sequence
-  (sequence_letter) @letter
-  (#eq? @letter "I")
-  (content) @markup.italic)
+  (sequence_letter) @character
+  (#eq? @character "L")
+  (content) @markup.uri)
```
Comment (the-mikedavis) [inline runtime/queries/pod/highlights.scm:62]:
```suggestion
  (content) @markup.link.url)
```

File: runtime/queries/pod/highlights.scm [hunk 6]
```diff
@@ -1,61 +1,72 @@
...
+  (#eq? @character "F")
+  (content) @markup.underline @string.special)
+
 (interior_sequence
-  (sequence_letter) @letter
-  (#eq? @letter "F")
-  (content) @markup.italic)
+  (sequence_letter) @character
+  (#eq? @character "I")
+  (content) @markup.emphasis)
```
Comment (the-mikedavis) [inline runtime/queries/pod/highlights.scm:57]:
```suggestion
  (content) @markup.bold)
```

File: runtime/queries/pod/highlights.scm [hunk 7]
```diff
@@ -1,61 +1,72 @@
...
+  ["<" ">"] @punctuation.delimiter
+)
 
 (interior_sequence
-  (sequence_letter) @letter
-  (#eq? @letter "B")
-  (content) @markup.bold)
+  (sequence_letter) @character
+  (#eq? @character "B")
+  (content) @markup.strong)
```
Comment (the-mikedavis) [inline runtime/queries/pod/highlights.scm:42]:
```suggestion
  (content) @markup.bold)
```

**Review by the-mikedavis — APPROVED** [review]
Thanks!

**Pull request merged**